src/rpc/types.rs

```diff
@@ -133,6 +165,11 @@ impl Rpc {
 
     /// Request blocknumber and return its value
     pub async fn block_number(&self) -> Result<u64, crate::rpc::types::RpcError> {
+        // skip sync check on strata rpc
+        if self.group == RouteGroup::StrataCL {
+            return Ok(0);
+        }
```

Comment:
This is an Ethereum RPC isn't it?  It's more correct to make this be `self.group != RouteGroup::Ethereum`.